tiles/src/main/java/com/protomaps/basemap/layers/Places.java

```diff
@@ -66,10 +86,13 @@ public void processNe(SourceFeature sf, FeatureCollector features) {
           theme_max_zoom)
         .setAttr("pmap:kind", kind)
         .setAttr("pmap:kind_detail", kind_detail)
-        .setAttr("population", sf.getString("pop_max"))
-        .setAttr("population_rank", sf.getString("rank_max"))
+        .setAttr("population", parseIntOrNull(sf.getString("pop_max")))
+        .setAttr("pmap:population_rank", parseIntOrNull(sf.getString("rank_max")))
```

Comment:
This is a "new" property for Protomaps schema v3 spec, but it's not OSMy so let's prefix it with `pmap:` here for consistency with other pmap specific properies... and in v4 we'd remove the prefix.